README.md

```diff
@@ -48,10 +48,8 @@ For more information see the [.NET Foundation Code of Conduct](CODE_OF_CONDUCT.m
 
 This project is supported by the [.NET Foundation](http://dotnetfoundation.org).
 
-## Contributors
+## 🎭 Contributors
```

Comment:
I don't like this emoji, specifically the fact one of the two faces has a sad expression.
I'd just use something like 🎉, to say "we're celebrating our contributors".

Reply:
I was looking for something with face or people and I thought this fit the bill as almost everyone would recognize this icon!

Reply:
How about these:

1. `🪄` - Magical Code
2. `🏆` - Code Champion
3. `🧭` - Leader (_who steers the project in the right direction_)
4. `🎊` - Celebrating the people behind! Also `🎉`!
5. `💯` - Always giving our best!

Reply:
I vote for number 2

Reply:
Number 2 is fine for me as well (same we used in the blog posts), but leave "Contributors" as the title.

Reply:
Title won't be changed. Those are just descriptions of the icons.

Reply:
Perfect, let's go with 🏆 then.